docs/features/software-templates/installation.md

```diff
@@ -189,17 +211,31 @@ docs on creating private GitHub access tokens is available
 Note that the need for private GitHub access tokens will be replaced with GitHub
 Apps integration further down the line.
 
+#### Github
+
 The Github access token is retrieved from environment variables via the config.
 The config file needs to specify what environment variable the token is
 retrieved from. Your config should have the following objects.
 
+#### Gitlab
+
+For Gitlab, we currently support the configuration of the GitLab publisher and
+allows to configure the private access token and the base URL of a GitLab
+instance:
+
 ```yaml
 scaffolder:
   github:
     token:
       $secret:
         env: GITHUB_ACCESS_TOKEN
     visibility: public # or 'internal' or 'private'
+  gitlab:
+    api:
+      baseUrl: https://gitlab.com
+      token:
+        $secret:
+          env: SCAFFOLDER_GITLAB_PRIVATE_TOKEN
```

Comment:
Does it make sense to add consistency in the name of these tokens for GitHub/GitLab?

`SCAFFOLDER_GITLAB_PRIVATE_TOKEN` can be `GITLAB_PRIVATE_TOKEN` to align with `GITHUB_ACCESS_TOKEN`

Reply:
Edit: `GITLAB_ACCESS_TOKEN` is more aligned.

Reply:
I was thinking that there could be a naming clash if we're using different access tokens for different accounts/plugins. That's something we can worry about later, though. Changed it to `GITLAB_ACCESS_TOKEN`. 👍